### ISSUE TITLE
Impossible to start creation of new card after success of first run

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#233]
In GitLab by @Kelson42 on Aug 25, 2018, 14:32

Created a first card successfully and then came back to the installer UI. Loading a new JSON and started again... It seems the software is blocked at the end stage of the precedent run. I only see in the log: `[STAGE 6/6: SD-card creation - 100%]`

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Aug 25, 2018, 14:33

Oh, using the « start new » button at the end of the process ?

**Comment by rgaudin** [comment]
In GitLab by @Kelson42 on Aug 25, 2018, 17:31

Yes, on the console I have the following error

```
Exception in thread Thread-1112:
Traceback (most recent call last):
  File "threading.py", line 920, in _bootstrap_inner
  File "threading.py", line 868, in run
  File "gui.py", line 1019, in target
  File "run_installation.py", line 24, in run_installation
  File "util.py", line 79, in stage
  File "util.py", line 72, in clean_up_stage
TypeError: unsupported operand type(s) for -: 'datetime.datetime' and 'NoneType'
```

**Comment by rgaudin** [comment]
In GitLab by @rgaudin on Aug 27, 2018, 18:17

closed via commit f48dc4874adf63579b675f77156b935e2705e3a5
